manifests/rbac/organization-controller-rbac.yaml

```
@@ -0,0 +1,80 @@
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+# Events for logging
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["create", "patch"]
+# ServiceAccounts in managed namespaces
+- apiGroups: [""]
+  resources: ["serviceaccounts"]
+  verbs: ["get", "list", "watch", "create", "update", "patch"]
+# Secrets in managed namespaces  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)